.github/workflows/static-checks.yaml

```diff
@@ -3,14 +3,18 @@ on:
   pull_request:
```

Comment:
shouldn't we enable it for the `pull_request` instead of `push` ?

Reply:
Do you mean only for pull_request? Because now it's enabled for both pull_request and push. 

Also this workflow is for all static checks, including go and manifest linting, so whatever we add here will apply to all static checks.